src/packages/arithmetic.rs

```diff
@@ -356,6 +362,19 @@ def_package!(crate:ArithmeticPackage:"Basic arithmetic", lib, {
         reg_sign!(lib, "sign", f64, f64);
     }
 
+    #[cfg(feature = "decimal")]
```

Comment:
In later versions I've changed the output of `sign` to be `INT` regardless of the type.  That's to avoid unnecessary boxed trait objects.  So therefore you can just return 1/-1/0 as `INT` and do away with `num_traits::One`.

Reply:
The num_traits::One usage has been removed, and replaced with straight 1, -1 literals.

Reply:
I've put in a comment: it should return 0 when x== 0.